### PR TITLE
Add additional human names

### PR DESCRIPTION
**Pull request by dorbarker**
**Content (Artwork / Missions / Jobs)**

## Summary
Adds some new variety for human vessels, and adjusts the phrase weights accordingly

### CODE REVIEW

File: data/human/names.txt [hunk 1]
```diff
@@ -674,6 +690,20 @@ phrase "male titles"
 		"Duke"
 		"Baron"
 		"Master"
+		"Brother"
```
Comment (mOctave) [inline data/human/names.txt:693]: Some of these are maybe a little too religious or obscure.

File: data/human/names.txt [hunk 2]
```diff
@@ -692,6 +728,19 @@ phrase "gender-neutral titles"
...
+		"Overseer"
+		"Cousin"
+		"Steward"
+		"Director"
+		"Prime Minister"
+		"General"
+		"Sergeant"
+		"Hajji"
+		"Rabbi"
+
```
Comment (mOctave) [inline data/human/names.txt:743]: Did you mean to add an extra line break here?

File: data/human/names.txt (hunk 1 above)
Comment (Anarchist2) [reply data/human/names.txt:693]: I think they're fine.

File: data/human/names.txt (hunk 2 above)
Comment (W1zrad) [reply data/human/names.txt:743]:
```suggestion
```

**Pull request merged**